plugins/modules/win_domain_ou.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_domain_ou
+short_description: Manage Active Directory Organizational Units
+author: Joe Zollo (@joezollo), Larry Lane (@gamethis)
```

Comment:
Might be best to set this as a list. You also need to add `version_added: 1.7.0` to denote when this was added

```suggestion
version_added: 1.7.0
author:
- Joe Zollo (@joezollo)
- Larry Lane (@gamethis)
```